test/e2e/generate_systemd_test.go

```diff
@@ -15,19 +15,13 @@ var _ = Describe("Podman generate systemd", func() {
 	It("podman generate systemd on bogus container/pod", func() {
 		session := podmanTest.Podman([]string{"generate", "systemd", "foobar"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).To(ExitWithError())
-	})
-
-	It("podman generate systemd bad restart policy", func() {
-		session := podmanTest.Podman([]string{"generate", "systemd", "--restart-policy", "never", "foobar"})
-		session.WaitWithDefaultTimeout()
-		Expect(session).To(ExitWithError())
+		Expect(session).To(ExitWithError(125, "foobar does not refer to a container or pod: no pod with name or ID foobar found: no such pod: no container with name or ID \"foobar\" found: no such container"))
 	})
 
 	It("podman generate systemd bad timeout value", func() {
 		session := podmanTest.Podman([]string{"generate", "systemd", "--time", "-1", "foobar"})
 		session.WaitWithDefaultTimeout()
-		Expect(session).To(ExitWithError())
+		Expect(session).To(ExitWithError(125, "invalid argument \"-1\" for \"-t, --time\" flag: strconv.ParseUint: parsing \"-1\": invalid syntax"))
```

Comment:
hint, if you like to avoid all the quoting you can use backticks to create a "raw" string without any required escaping, i.e.
```
`invalid argument "-1" for "-t, --time" flag: strconv.ParseUint: parsing "-1": invalid syntax`
```

Not required of course but IMO makes it easer to read.

Reply:
Thank you! I prefer the backtick form too, but its use in our code base is rare. I'm in the process of fixing these (only in my changes) and will repush.